app/components/TrackComponent/index.js

```diff
@@ -0,0 +1,79 @@
+/**
+ *
+ * TrackComponent
+ *
+ */
+
+import React, { memo } from 'react';
+import PropTypes from 'prop-types';
+import styled from 'styled-components';
+import { isEmpty } from 'lodash';
+import { colors } from '@app/themes/index';
+import If from '@components/If';
+import { Card, Image, Typography, Button } from 'antd';
+
+const { Title, Paragraph } = Typography;
+
+const TrackCardContainer = styled(Card)`
+  && {
+    border-radius: 0.5rem;
+    width: ${props => (props.width ? props.width : '25rem')};
+    border: none;
+    background-color: ${colors.secondary};
+    text-align: center;
+`;
+
+const StyledDescription = styled.div`
+  display: flex;
+  flex-direction: column;
+`;
+
+const PlayTrackBtn = styled(Button)`
+  && {
+    background-color: ${colors.btnBackground};
+    border: none;
+    color: ${colors.btnText};
```

Comment:
can we name the color and use it, instead of naming it element specific like btnText

Reply:
updated the changes